src/Symfony/Component/Webhook/Server/Transport.php

```diff
@@ -37,6 +40,8 @@ public function send(Subscriber $subscriber, RemoteEvent $event): void
         $this->body->configure($event, $subscriber->getSecret(), $options);
         $this->signer->configure($event, $subscriber->getSecret(), $options);
 
-        $this->client->request('POST', $subscriber->getUrl(), $options->toArray());
+        $response = $this->client->request('POST', $subscriber->getUrl(), $options->toArray());
+
+        $this->eventDispatcher?->dispatch(new WebhookSentEvent($subscriber, $event, $response));
```

Comment:
I'm not sure to understand your point
Given the event has the Response object, whenever someone will ask for (for example) the status, the response will be fulfilled, am I right?

Reply:
Well, in the existing code, the Transport still waits until the response headers are available (as it does not keep track of the response, its `__destruct` method is called by the time the `request` method returns it, and the `__destruct` method enforces the error handling behavior). this is not a fire-and-forget call for which we hope it will work.

Reply:
Ok, thank you for this correction !